storybookjs/storybook
# UI: Add new Radix toolbar to the Core UI library

## What I did

The toolbar is one of the main piece of the Storybook manager. We would like to improve it to be more accessible and well documented. In this PR we created a new component `Toolbar />` as part on the new Core UI library. It is based on Radix Toolbar and extend all props with our custom styling. This new component will replace `FlexBar` and could be used on the manager as well as addons for multiple use cases.

Older IconButton are still working but as part of this PR, I'm updating most of the old IconButton with the new ones to make it coherent with the rest of the manager.

## How to test

1. Run this branch locally using `yarn storybook:ui`
2. Open Storybook in your browser at this link http://localhost:6006/?path=/story/core-ui-toolbar--base
3. You should now see the new Toolbar component

## Checklist



      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

#### Maintainers


`["cleanup", "BREAKING CHANGE", "feature request", "bug", "build", "documentation", "maintenance", "dependencies", "other"]`



### 🦋 Canary release



This PR does not have a canary release associated. You can request a canary release of this pull request by mentioning the `@storybookjs/core` team here.

_core team members can create a canary release [here](https://github.com/storybookjs/storybook/actions/workflows/canary-release-pr.yml) or locally with `gh workflow run --repo storybookjs/storybook canary-release-pr.yml --field pr=<PR_NUMBER>`_
